include/vara/Feature/XmlParser.h

```diff
@@ -34,6 +35,7 @@ class XmlParser {
   void parseOptions(xmlNode *N, bool Num);
   void parseConstraints(xmlNode *N);
   void parseVm(xmlNode *N);
+  static std::pair<int, int> parseLocation(xmlNode *N);
```

Comment:
Hmm, is there a reason why you made this one static and the other methods are not static?